app/src/theme/index.js

```diff
@@ -88,7 +88,7 @@ const theme = {
     alertHover: '#EF5350',
     borderColorDark: '#9FA8DA',
     darkGray: '#acacac',
-    error: '#fa0000',
+    error: '#e57373',
```

Comment:
also there is this color in the theme already so instead of changing color itself import different one ....as you can see above there is 'alert' color which is the same as one you need